wordpress-mobile/WordPress-iOS
# Fix unit tests in Xcode 9

Replace OCMock on managed objects with the TestContextManager. Refactored some remotes so they take site IDs instead of Blog objects. Added ModelTestHelper to ensure test Blog/WPAccount objects have all the required properties filled.

See #7343 

To test:

- Make sure tests pass in Xcode 8
- Make sure tests pass in Xcode 9

Needs review: @astralbodies
